src/message.rs

```diff
@@ -113,13 +126,17 @@ impl NetfilterMessage {
     pub fn subsys(&self) -> u8 {
```

Comment:
If possible, create another patch to change this `u8` to a `enum`.

Otherwise the user of this crate still need  to read kernel header to understand this meaning of this `u8`.